spec/support/rake_helper.rb

```diff
@@ -0,0 +1,17 @@
+require 'rake'
+
```

Comment:
One way of doing this differently instead of loading your rake tasks in isolatation is you can just load all them in before your test suite? like:

```ruby
RSpec.configure do |config|
  
  config.before(:suite) do
    Rails.application.load_tasks
  end

end
```

But regardless the issue with the way we were previously doing this was we were constantly reloading the rake files over and over again resulting in a ton of spam to the logs for overwriting constatns that were already defined.